tdp/cli/commands/conftest.py

```diff
@@ -1,11 +1,37 @@
 # Copyright 2022 TOSIT.IO
 # SPDX-License-Identifier: Apache-2.0
 
+from collections.abc import Generator
 from pathlib import Path
+from typing import Any
 
 import pytest
+from click.testing import CliRunner
+from sqlalchemy import create_engine
 
+from tdp.cli.commands.init import init
 from tdp.conftest import generate_collection_at_path
+from tdp.core.models import BaseModel
+
+
+@pytest.fixture
+def tdp_init(
+    collection_path: Path, db_dsn: str, vars: Path
+) -> Generator[list[Any], Any, None]:
+    base_args = [
+        "--collection-path",
+        collection_path,
+        "--database-dsn",
+        db_dsn,
+        "--vars",
+        str(vars),
```

Comment:
Why casting only the `vars` while `collection_path` is also a Path?

Either don't cast any or cast both to return a `list[str]`

Reply:
Because we mostly casted the `vars` before while not casting the `collection_path`. 
However, it works if we do not cast any. I do not know the reason why the `vars` were casted before in some tests, I though there was a specific reason. I will just remove it.